### PR TITLE
Update interaction docs

### PR DESCRIPTION
**Pull request by niktre**
The major parts of the PR are:
- Introduced a division into various interaction types for user guide (Angles, Bonds, Pair, etc.)
- Remove several Unique potentials
- Add docs for FENE, FENECapped, AngularHarmonic, AngularCosineSquared

### CODE REVIEW

File: src/interaction/FENECapped.py [hunk 1]
```diff
@@ -24,64 +24,65 @@
...
 
-.. function:: espressopp.interaction.FixedPairListFENECapped.setFixedPairList(fixedpairlist)
+**Example of usage**
 
-		:param fixedpairlist: 
-		:type fixedpairlist: 
+>>> Please, refer to the example of FENE potential
 
-.. function:: espressopp.interaction.FixedPairListFENECapped.setPotential(potential)
+Go to FENE-example :py:class:`espressopp.interaction.FENE`
```
Comment (jkrajniak) [inline src/interaction/FENECapped.py]: what is FENE-example?

Comment (niktre) [reply src/interaction/FENECapped.py]: it refers to the example on FENE-potential usage. Should I change the formulation here?

Comment (jkrajniak) [reply src/interaction/FENECapped.py]: Does `:py:class:`espressopp.interaction.FENE` convert to a link ?

Comment (jkrajniak) [reply src/interaction/FENECapped.py]: Good :+1:

**Pull request merged**